index.js

```diff
@@ -22,7 +22,11 @@ exports.init = function(opts) {
   exports.SchemaAdapter = require('./adapters/mongoose');
 
   exports.getModels = function () {
-    return opts.mongoose.models;
+    if ((typeof opts.getModels) === 'function') {
```

Comment:
Ok I can change it to `included_models` as an array of names.
Do we also go for `excluded_models` which if present, filters `opts.mongoose.models` negatively with names in this array as well ?
Do we limit ourselves to the scope of the `getModels` function or do we go beyond? In the rails liana, the `SchemaUtils.model_included` is used all over the place, similarly here shall we use a function to replace all usages of `opts.mongoose.models`?